dali/python/nvidia/dali/external_source.py

```diff
@@ -73,10 +75,13 @@ def __init__(self, callback, is_multioutput, instances = [], cuda_stream = None,
         self.batch = batch
         self.current_iter = 0
         self.current_sample = 0
```

Comment:
Do we use that variable anyhow?

Reply:
Yeah, in per-sample mode it is used to calculate SampleInfo.idx_in_epoch.